core/templates/dev/head/pages/topics_and_skills_dashboard/TopicsAndSkillsDashboard.js

```diff
@@ -21,6 +21,9 @@ oppia.constant(
 oppia.constant('EVENT_TOPIC_DELETED', 'topicDeleted');
 oppia.constant('EVENT_TYPE_TOPIC_CREATION_ENABLED', 'topicCreationEnabled');
 
+oppia.constant(
+  'TOPIC_EDITOR_STORY_URL_TEMPLATE', '/topic_editor_story_handler/<topic_id>');
```

Comment:
Where is this used?

Reply:
I am importing EditableTopicBackendApiService in topics_and_skills_dashboard.html, for delete topic, and that uses this.

Reply:
But you should not have the same constant declared in two places in the codebase -- perhaps move it out to some higher top-level file.

Reply:
Done, moved it to EditableTopicBackendApiService.js itself, since it's not used anywhere else.